tests/NLog.UnitTests/Mocks/AppEnvironmentMock.cs

```diff
@@ -32,23 +32,43 @@
 // 
 
 using System;
+using System.Collections.Generic;
+using System.Xml;
 using NLog.Internal.Fakeables;
 
 namespace NLog.UnitTests.Mocks
 {
-    public class FileMock : IFile
+    internal class AppEnvironmentMock : IAppEnvironment
```

Comment:
FYI, you could  use NSubsitute  for this when #3274 is merged

Reply:
Fine with implementing simple mocks by hand. It is more when meeting the monster-interface (or non-interfaces) that Mock-frameworks are helpful.